src/app/main/component/shared/components/calendar-base/calendar-base.component.spec.ts

```diff
@@ -0,0 +1,299 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { TranslateService } from '@ngx-translate/core';
+import { CalendarBaseComponent } from './calendar-base.component';
+import { EventEmitter, Injectable } from '@angular/core';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { MatDialogModule } from '@angular/material/dialog';
+import { CalendarInterface } from '@global-user/components/profile/calendar/calendar-interface';
+import { LanguageService } from 'src/app/main/i18n/language.service';
+
+@Injectable()
+class TranslationServiceStub {
+  public getTranslation = new EventEmitter<any>();
+  public onDefaultLangChange = new EventEmitter<any>();
+  public getDefaultLang = new EventEmitter<any>();
+  public unsubscribe = new EventEmitter<any>();
+}
+
+describe('CalendarBaseComponent', () => {
+  let component: CalendarBaseComponent;
+  let fixture: ComponentFixture<CalendarBaseComponent>;
+  let calendarMock: CalendarInterface;
+  let isMonthCalendar: boolean;
```

Comment:
let's be consistent and have 1 pattern for branch naming - should be bugfix/#2809-unit-test-for-calendar-component

Reply:
also, provide PR description